markdown/dev/reference/api/attributes/en.md

```diff
@@ -12,6 +12,12 @@ Attributes new Attributes()
 
 The Attributes constructor takes no arguments.
 
+## Properties
+
+An Attributes object comes with the following property:
+
+- `list` : The of individual attributes.
```

Comment:
Is there a word missing here?

Reply:
There is. List holds the internal object in which attributes are stored. So if we're documenting it, we should fix the description.